### ISSUE TITLE
Whitelist yaml detection when using SafeLoader

### ISSUE THREAD

**Issue by mschwager** [dlint-py/dlint#25]
The following `pyyaml` calls should be safe:

```python
yaml.load(..., Loader=yaml.SafeLoader)
yaml.load(..., Loader=yaml.CSafeLoader)
```

I believe this is equivalent to using `safe_load`, but I've encountered a few false positives in the wild using this code.
